rightmesh/contracts/Timelock.sol

```diff
@@ -32,23 +32,30 @@ contract Timelock is Ownable {
   function Timelock(ERC20Basic _token, uint256 _startTime, uint256 _cliffDuration, uint256 _cliffReleasePercent, uint256 _gradualDuration, uint256 _gradualReleasePercentage) public {
 
     // sanity checks
-    assert(_cliffReleasePercent.add(gradualReleasePercentage) <= 100);
+    assert(_cliffReleasePercent.add(_gradualReleasePercentage) <= 100);
     assert(_startTime > now);
 
+    allocationFinished = false;
+
     token = _token;
     startTime = _startTime;
     cliffDuration = _cliffDuration;
     cliffReleasePercentage = _cliffReleasePercent;
     gradualDuration = _gradualDuration;
     gradualReleasePercentage = _gradualReleasePercentage;
+  }
 
-    // allocatedTokens[0x14723a09acff6d2a60dcdf7aa4aff308fddc160c] = 10 ether;
+  function allocateTokens(address _address, uint256 _amount) onlyOwner public returns (bool) {
+    require(!allocationFinished);
 
```

Comment:
assume this function is the method for setting the vesting token for employees. We shouldn't allow the amount is larger than the total amount of tokens which the current Timelock contract is holding.

Reply:
After discussion, the workflow will be:

1. allocate tokens to employees first.
2. then mint tokens according to the total amount of tokens, which has been allocated.